assets/l10n/app_en.arb

```diff
@@ -483,5 +483,24 @@
   "notifSelfUser": "You",
   "@notifSelfUser": {
     "description": "Display name for the user themself, to show after replying in an Android notification"
+  },
+  "pollWidgetQuestionMissing": "(Untitled Poll)",
+  "@pollWidgetQuestionMissing": {
+    "description": "Text to display for a poll when the question is missing"
+  },
```

Comment:
Not sure if this should be "(Untitled poll)" or something else; @alya might have thoughts on this.

Reply:
"(no title)" might be an option (inspired by "(no topic)")

Reply:
I think it's helpful to include "poll", so it's clear what kind of title-lacking thing we're rendering.